### PR TITLE
Define doctype method on XMLParser target

### PR DESCRIPTION
**Pull request by mvdbeek**
That should silence some Deprecation Warnings.

### CODE REVIEW

File: lib/tool_shed/util/xml_util.py [hunk 1]
```diff
@@ -82,18 +85,10 @@ def parse_xml(file_name):
         return None, "File does not exist %s" % str(file_name)
 
     with open(file_name, 'r') as fobj:
-        if using_python_27:
```
Comment (nsoranzo) [inline lib/tool_shed/util/xml_util.py:85]: You can now remove the definition of `using_python_27` at line 13.

**Review by nsoranzo — APPROVED** [review]

**Comment by mvdbeek** [comment]
Hmm, looks like there's one left:
`  /usr/lib/python2.7/xml/etree/ElementTree.py:1638: DeprecationWarning: This method of XMLParser is deprecated.  Define doctype() method on the TreeBuilder target.
`

Could be coming from one of the dependencies though.

**Comment by nsoranzo** [comment]
@mvdbeek https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/util/__init__.py#L219

**Comment by mvdbeek** [comment]
Hmm, that does define `doctype`, so I think it should be fine ?
```
import warnings
warnings.simplefilter('always', DeprecationWarning)
from galaxy.util import parse_xml
parse_xml('../config/tool_conf.xml')
```
doesn't report any warning

**Pull request merged**